content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -839,33 +668,24 @@ node {
 
 ===== Combining credentials in one step
 
-Using the *Snippet Generator*, you can make multiple credentials available
-within a single `withCredentials( ... ) { ... }` step by doing the following:
+Using the *Snippet Generator*, you can make multiple credentials available within a single `withCredentials( ... ) { ... }` step by doing the following:
 
-. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click
-  the name of your Pipeline project/item.
-. On the left, click *Pipeline Syntax* and ensure that the *Snippet Generator*
-  link is in bold at the top-left. (If not, click its link.)
-. From the *Sample Step* field, choose *withCredentials: Bind credentials to
-  variables*.
+. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click the name of your Pipeline project/item.
```

```suggestion
. From the Jenkins Dashboard, select the name of your Pipeline project/item.
```
Suggestion to use Jenkins Dashboard and replace `click` with `select` as per previous comment.